apps/web/config/routes.rb

```diff
@@ -4,3 +4,5 @@
 
 resources :users, only: %i[new create]
 resources :sessions, only: %i[new create]
+
+delete 'sessions', as: 'logout'
```

Comment:
Warden provides a session_id, which is all that's needed for this to be added to the resources array. Can you not just generate the link with that?

Reply:
This would feel strange to me as we we're not actually using the session_id in the controller action - instead we're just invoking the warden logout method to destroy the active session. This means that we'd be treating the session_id as the resource identifier, but not actually identifying any resource with it - meaning that any arbitrary value could passed in the URL. Personally I can't really see the value of doing that just for the sake of making it a member route and being able to include it in the resources array - unless I have misunderstood your suggestion? I think in this instance having a DELETE /sessions route as we've done here is okay. Happy to discuss further though if you disagree

Reply:
See your point, will resolve this.